presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/StandardDebugger.java

```diff
@@ -0,0 +1,29 @@
+package io.prestosql.tests.product.launcher.env;
+
+import com.google.common.base.Splitter;
+
+import java.util.Optional;
+
+import static java.util.Locale.ENGLISH;
+
+public enum StandardDebugger
+{
+    PRESTO,
+    // METASTORE, // TODO
```

Comment:
```
export HADOOP_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5006"
cat /usr/lib/hive/bin/hive | grep 'exit 5' -v > x; mv -f x /usr/lib/hive/bin/hive
supervisorctl stop hive-metastore
chmod +x /usr/lib/hive/bin/hive
su hive -c 'hive --service metastore --hiveconf hive.root.logger=DEBUG,console --hiveconf hive.log.threshold=DEBUG' -s /bin/bash
```

Reply:
the correct way is using `hive --debug=....` option.